src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public Figure getRandomFigure() {
+        Random random = new Random();
+        int index = random.nextInt(RandomFigure.values().length);
+        RandomFigure randomFigure = RandomFigure.values()[index];
+        Figure figure;
+        String randomColor = new ColorSupplier().getRandomColor();
+        switch (randomFigure) {
+            case CIRCLE:
+                double randomRadius = random.nextDouble();
+                figure = new Circle(randomRadius, randomColor);
+                break;
+            case SQUARE:
+                double randomSide = random.nextDouble();
+                figure = new Square(randomSide, randomColor);
+                break;
+            case RECTANGLE:
+                double randomFirstSide = random.nextDouble();
+                double randomSecondSide = random.nextDouble();
+                figure = new Rectangle(randomFirstSide, randomSecondSide, randomColor);
+                break;
+            case RIGHTTRIANGLE:
+                double randomFirstLeg = random.nextDouble();
+                double randomSecondLeg = random.nextDouble();
+                figure = new RightTriangle(randomFirstLeg, randomSecondLeg, randomColor);
+                break;
+            case ISOSCELESTRAPESOID:
+                double randomLittleSide = random.nextDouble();
+                double randomBigSide = random.nextDouble();
+                double randomAltitude = random.nextDouble();
+                figure = new IsoscelesTrapezoid(randomLittleSide, randomBigSide,
+                        randomAltitude, randomColor);
+                break;
+            default:
+                figure = getDefaultFigure();
+                figure.color = Color.values()[random.nextInt(Color.values().length)].name();
+                break;
+        }
+        return figure;
+    }
+
+    public static Figure getDefaultFigure() {
+        final double defaultRadius = 10.0;
+        final String color = Color.WHITE.name();
+        Circle circle = new Circle(defaultRadius, color);
+        return circle;
+    }
```

Comment:
method should not be a static;
to create constants;
do not create useless variable
```suggestion
    public Figure getDefaultFigure() {
        return new Circle(CIRCLE_RADIUS, Color.WHITE.name());
    }
```